src/BioFSharp.Mz/ProteinInference.fs

```diff
@@ -30,6 +36,68 @@ module ProteinInference =
             Class: PeptideEvidenceClass
         }    
 
+    /// For a group of proteins, contains information about all peptides that might be used for its quantification and score calculated for it.
+    type InferredProteinClassItemScored =
+        {
+            GroupOfProteinIDs: string
+            PeptideSequence  : string []
+            Class            : PeptideEvidenceClass
+            TargetScore      : float
+            DecoyScore       : float
+            Decoy            : bool
+            DecoyBigger      : bool
```

Comment:
DecoyBigger can probably be renamed

Reply:
I can't think of another name for that, since it describes exactly what it contains.

Reply:
Changed name to DecoyHasBetterScore